cmd/version.go

```diff
@@ -33,7 +33,7 @@ var versionCmd = &cobra.Command{
 	Short: "Show application version",
 	Long:  `Show the application version and exit.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("k6 v" + consts.Version)
+		fmt.Println("k6 v" + consts.GetFullVersion())
```

Comment:
Would it make sense to drop the "v" prefix here, as @cuonglm mentioned over Slack?

It's already a prefix on the Git version tags, and if we ever wanted to read those (see my other comment), then it would have to be stripped to avoid the duplicate "vv".

Personally, I wouldn't mind getting rid of the "v" prefix entirely, even in Git tags.

Reply:
the `v` in the tags is part of [how go modules know](https://github.com/golang/go/wiki/Modules#modules)(last paragraph of the linked section) that that tag is for a version ;) So we shouldn't remove them from the tags. 

On the other hand removing it from this message is a good idea IMO as well ...

Reply:
Yeah, we can't and shouldn't remove it from the git tags. And if we were implementing `k6 version` from scratch, I probably also wouldn't have included the `v` in its output. But given that the current output includes `v`, I'm not sure I want to change that for such a minor improvement, as not to break code that potentially parses this value. 

Even the addition of extra text after the version that the current PR introduced makes me weary, but I see no way around that, whereas AFAIK there's nothing essentially wrong with including the `v` in the output, i.e. it's a matter of preference.